writing-apps/our-first-app/the-build-system.md

```diff
@@ -59,8 +59,8 @@ Change into the build directory and use `ninja` to build. Then, if the build is
 
 ```bash
 cd build
-ninja
-ninja install
+sudo ninja
+sudo ninja install
```

Comment:
You definitely don't need to build as root, and `ninja install` will automatically use polkit to handle elevating privileges.